### PR TITLE
converted JSX component to Pascal Case

### LINKED ISSUE #38850: Pathnames are case sensitive
`/learn` is not the same as `/LEARN` or `/LeArN`. The second two take you to user profiles, I believe they are the same user profiles. Just starting an issue about it, I haven't looked into it at all.

### PR DESCRIPTION
**Pull request by kailaa-sri**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

In ToggleSetting.js and ToggleButton.js changed the imported JSX to Pascal case

### CODE REVIEW

**Review by raisedadead — CHANGES_REQUESTED** [review]
Did you mean to refer to `#38850` intentionally?

**Comment by kailaa-sri** [comment]
> Did you mean to refer to `#38850` intentionally?
image

No but it resolves these warnings.

File: client/src/components/helpers/ToggleButton.js [hunk 1]
```diff
@@ -1,74 +1,78 @@
 import React, { Fragment } from 'react';
 import PropTypes from 'prop-types';
 import {
-  ToggleButtonGroup as BSBG,
-  ToggleButton as TB
+    ToggleButtonGroup as bsbg,
+    ToggleButton as tb
```
Comment (ojeytonwilliams) [inline client/src/components/helpers/ToggleButton.js:5]:
```suggestion
    ToggleButtonGroup,
    ToggleButton as BootstrapToggleButton
```
How about this?  It's a little verbose, but bsbg and tb don't mean anything unless you look them up or already know what they mean.

**Review by ojeytonwilliams — CHANGES_REQUESTED** [review]
Could you apply `npm run format` to this?  I'm not sure why the commit hook failed, but this won't pass the linting step.

**Comment by ojeytonwilliams** [comment]
@kailaa-sri you're welcome to push those changes as a commit to this PR if you'd like.  There's no need to open a new one.

**Comment by raisedadead** [comment]
> No but it resolves these warnings.

Could you remove the references to the unrelated issue. You can edit your last commit after you have made changes with `git commit --amend` and push to the same PR branch with `git push origin <branch name> --force`

Thanks.

**Comment by raisedadead** [comment]
Please reopen a fresh PR when ready.

**Pull request closed without merging**